### DISCUSSION TITLE
how to enable the udp relay, the config options of udp relay is disabled by default.

### DISCUSSION THREAD

**Discussion by LaiZhou** [shadowsocks/shadowsocks-android#2725] [Q&A]
how to enable the udp relay, the config options of udp relay is disabled by default.

**Comment by madeye** [comment]
UDP relay is always enabled, you don't need to enable it manually.

**Reply by imReker** [reply]
> UDP relay is always enabled, you don't need to enable it manually.

Sorry, but why UDP relay is DISABLED in profile config. And there are 1 star comments under Shadowsocks page in Play Store.

**Reply by madeye** [reply]
There's no such option.

Make sure you're using the latest version.
